tests/components/ecobee/test_humidifier.py

```diff
@@ -43,6 +44,8 @@ async def test_attributes(hass: HomeAssistant) -> None:
 
     state = hass.states.get(DEVICE_ID)
     assert state.state == STATE_ON
+    if state.attributes.get(ATTR_CURRENT_HUMIDITY):
```

Comment:
We don't want conditions in tests. Please parametrize the test instead.

Reply:
Thnx, missed that one. You are right: https://github.com/home-assistant/core/pull/115197